backup.sh

```diff
@@ -63,11 +76,59 @@ cleanup() {
     printf "\n"
 }
 
+# export metrics for prometheus/node_exporter textfile collector
+# metrics: 
+#   - date of last backup
+#   - backup file sizes
+#   - duration
+metrics_dump() {
+    printf "Writing backup metrics to file ${NODE_EXPORTER_DIR}/backup_full.prom\n"
+
+    cat << EOF > "${NODE_EXPORTER_DIR}/backup_full.prom.$$"
+# HELP backup_last_time_seconds system time of last backup in seconds
+# TYPE backup_last_time_seconds counter
+backup_last_time_seconds $END
```

Comment:
Could this also export the available size on disk ?

Reply:
sure, but I think the standard node_exporter metrics already provide that